[IMP] rental: updated manage deposits to 19.1 (#16012)
odoo/documentation: content/applications/sales/rental/manage_deposits.rst

Manage deposits
===============

Requiring fixed deposits is common in many rental scenarios, such as collecting security deposits.
Security deposits are insurance in case of damage, theft, or unpaid fees. It is typically refunded
when the rented product is returned in good condition, and all terms of the rental or service
agreement are met.

This document covers the standard options for :ref:`configuring <rental/manage_deposits/config>`,
:ref:`collecting <rental/manage_deposits/optional-product>`, and :ref:`refunding
<rental/manage_deposits/refund>` deposits within the **Rentals** app.

App integration configuration
=============================

The following apps are essential for configuring deposit service products and processing deposit
refunds:

- **Sales** app: Enables the :guilabel:`Price` tab on product forms when
  :doc:`../sales/products_prices/prices` is enabled. Also enables the use of quotation templates.
- **Accounting** app: Enables the use of :doc:`Credit Notes

>   <../../finance/accounting/customer_invoices/credit_notes>` on rental orders, which are required

Reviewer: The 'manage your pricing' link just takes me to a menu with 3 options. Typically, I would expect to see the info mentioned directly. Where on a product form (or on the App settings page) would I find the option to enable it? I recommend having that be the reference doc (preferably an anchor to the spot that explicitly explains enabling the option)

  when returning a deposit.

.. _rental/manage_deposits/config:

Configuration
=============

> For every rental product that requires a deposit, create a *deposit service product* representing

Reviewer: ```suggestion
For every rental product that requires a deposit, a second product **must** be configured, specifically for the deposit.

Example:
When configuring a digital camera rental that requires a deposit, two products must be configure: a `Digital Camera Rental` and a `Digital Camera Rental Deposit` product. 
```

I woudl be super-specific that each product needs 2 products (since that's not super intuitive, and took me a econd to figure it out(. Then instead of a tip, you cna add the example right after, so clearly illustrate the point. - Just a suggesiton!

product:

- :ref:`Manually <rental/service_products/create-rental-order>`
- :ref:`rental/manage_deposits/optional-product`
- :ref:`rental/manage_deposits/use-quotation-template`

.. _rental/manage_deposits/optional-product:

Adding deposits through the optional product menu
-------------------------------------------------

To :ref:`configure an optional product <sales/sales_quotations/config-optional-product>` for a
deposit, select the desired product or create one. On the product's form, click the
:guilabel:`Sales` tab and add the :ref:`deposit <rental/manage_deposits/config>` to the
:guilabel:`Optional Products` field in the *Upsell & Cross-sell* section.

.. note::
   If :ref:`creating a new product <rental/physical-products>`, be sure to configure the rental
   product's prices on the :guilabel:`Prices` tab.

.. image:: manage_deposits/sales-tab-optional-product.png
   :alt: A Digital Camera rental product form with an Optional Product field configured to
    its deposit.

Then create a :ref:`rental order <rental/order>` and add the rental product in the :guilabel:`Order
Lines` tab. A *Configure your product* pop-up window will appear. Click :icon:`fa-plus`

> :guilabel:`Add` for the deposit item, then click :guilabel:`Confirm`.

Reviewer: ```suggestion
Then create a :ref:`rental order <rental/order>` and add the rental product in the :guilabel:`Order
Lines` tab. A *Configure your product* pop-up window appears. Click :icon:`fa-plus`
:guilabel:`Add` for the deposit item, then click :guilabel:`Confirm`.
```
I can't remember the where or the why, but we avoid using "will" (usually the linter catches it) so we try to keep everything in the present tense.

.. image:: manage_deposits/configure-your-product-window.png
   :alt: A Configure your product window for a Digital Camera and its deposit.

.. _rental/manage_deposits/use-quotation-template:

Using a quotation template
--------------------------

Using quotation templates helps tailor the product list to the customer while ensuring the necessary
deposits are included in the order. Navigate to :menuselection:`Rental app --> Configuration -->
Quotation Templates` and click :guilabel:`New` to get started.

:ref:`Create a quotation template <sales/send_quotations/create_templates>` and then create a
:ref:`rental order <rental/order>`. Select the desired template in the :guilabel:`Quotation
Template` field.

Process a deposit on a rental order
===================================

Once the rental order is confirmed by the customer, the deposit should be invoiced either before or
at the time the customer picks up the rented product.

To do so, change the :guilabel:`Delivered` column for the deposit service product to `1`. The text
color changes from black to blue, indicating that the line is ready for invoicing.

.. image:: manage_deposits/digital-camera-deposit-item-line.png

>    :alt: Digital Camera Deposit on a rental order with the Delivered column set to 1.

Reviewer: Question - does the rental deposit product have ot have a time frame? I assume it wouldn't since it's tied to the product, and just looks odd ot see it.  basically, can you configure the deposit ot not show the time?

Click :guilabel:`Create Invoice` and select :guilabel:`Regular invoice` from the *Create invoice(s)*
pop-up window. Click :guilabel:`Create Draft` to generate the invoice. After verifying that the
draft is correct, click :guilabel:`Confirm`, then click :guilabel:`Send` to email the invoice to the
customer.

.. image:: manage_deposits/invoice-for-deposit.png
   :alt: A confirmed invoice for a rental deposit.

For in-person transactions, click :guilabel:`Pay`, verify the *Pay* pop-up window,then click

> :guilabel:`Create Payment` and click Send to email a copy of the receipt.

Reviewer: ```suggestion
For in-person transactions, click :guilabel:`Pay`, verify the *Pay* pop-up window, then click
:guilabel:`Create Payment` and click Send to email a copy of the receipt.
```

.. _rental/manage_deposits/refund:

Refund deposits upon return
===========================

To process a deposit return, the **Accounting** app must be installed. When a customer :ref:`returns
the rented item <rental/return-products>`, begin the deposit return by preparing a :doc:`credit note
<../../finance/accounting/customer_invoices/credit_notes>` against the original deposit invoice.

First, go to the :menuselection:`Rental app --> Orders --> Orders`, and select the correct rental

> order. Click the :guilabel:`Invoices` smart button, find the invoice for the deposit, and create a

Author: @larm-odoo Which icon are you referring to?

:doc:`credit note <../../finance/accounting/customer_invoices/credit_notes>`.

.. image:: manage_deposits/paid-deposit-invoice.png
   :alt: A paid deposit invoice to show the Credit Note button.

.. tip::
   In the *Credit Note* pop-up window, enter `Security Deposit Refund` in the :guilabel:`Reason

>    displayed on the Credit Note` field.

Reviewer: ```suggestion
   displayed on Credit Note` field.
```

order <rental/order>`. Select the desired template in the :guilabel:`Quotation Template` field.

Process a deposit on a rental order
===================================

Once the rental order is confirmed by the customer, the deposit should be invoiced either before or
at the time the customer picks up the rented product.

To do so, change the :guilabel:`Delivered` column for the deposit service product to `1`. The text
color changes from black to blue, indicating that the line is ready for invoicing.

.. image:: manage_deposits/digital-camera-deposit-item-line.png
   :alt: Digital Camera Deposit on a rental order with the Delivered column set to 1.

Click :guilabel:`Create Invoice` and select :guilabel:`Regular invoice` from the *Create invoice(s)*
pop-up window. Click :guilabel:`Create Draft` to generate the invoice. After verifying that the
draft is correct, click :guilabel:`Confirm`, then click :guilabel:`Send` to email the invoice to the
customer.

.. image:: manage_deposits/invoice-for-deposit.png
   :alt: A confirmed invoice for a rental deposit.

For in-person transactions, click :guilabel:`Pay`, verify the *Pay* pop-up window, then click

> :guilabel:`Create Payment` and click Send to email a copy of the receipt.

Reviewer 2: ```suggestion
:guilabel:`Create Payment` and click :guilabel:`Send` to email a copy of the receipt.
```

.. _rental/manage_deposits/refund:

Refund deposits upon return
===========================

To process a deposit return, the **Accounting** app must be installed. When a customer :ref:`returns
the rented item <rental/return-products>`, begin the deposit return by preparing a :doc:`credit note
<../../finance/accounting/customer_invoices/credit_notes>` against the original deposit invoice.

First, go to the :menuselection:`Rental app --> Orders --> Orders`, and select the correct rental
order. Click the :icon:`fa-pencil-square-o` :guilabel:`Invoices` smart button, find the invoice for
the deposit, and :ref:`issue a credit note <accounting/credit_notes/issue-credit-note>`.

.. image:: manage_deposits/paid-deposit-invoice.png
   :alt: A paid deposit invoice to show the Credit Note button.

.. tip::
   In the *Credit Note* pop-up window, enter `Security Deposit Refund` in the :guilabel:`Reason
   displayed on Credit Note` field.

   .. image:: manage_deposits/credit-note-pop-up-window.png
      :alt: Credit Note pop-up window with Security Deposit Refund entered in the Reason field.

Return to the rental order using the :icon:`fa-pencil-square-o` :guilabel:`Sales Orders` smart

> button. Set the *delivered quantity* for the deposit to `0`. This step ensures the deposit is marked

Reviewer 2: are we referring to a field on the SO here? If so, let's use guilabel!
